src/test/llm/llmnode_test.cpp

```diff
@@ -2841,7 +3310,14 @@ TEST_F(LLMConfigHttpTest, LLMNodeWorkspacePathToFileNotDir) {
     ASSERT_EQ(mediapipeDummy.validate(manager), StatusCode::LLM_NODE_DIRECTORY_DOES_NOT_EXIST);
 }
 
-TEST_F(LLMConfigHttpTest, LLMNodeResourceInitFailed) {
+class LLMConfigHttpTestParameterized : public ::testing::Test, public ::testing::WithParamInterface<std::tuple<std::string, ovms::StatusCode>> {
+public:
+    void SetUp() { py::initialize_interpreter(); }
```

Comment:
are those the new tests that need interpreter? if so, we should modify run_unit_tests.sh to run them in separate processes